### PR TITLE
Update maps

### PR DESCRIPTION
**Pull request by tylerferrara**
gestureFX added to every map (except room booking)
Image caching is done one time only at Main method -> big speed boost

### CODE REVIEW

File: exclusionBugs.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <FindBugsFilter>
   <Match>
-    <Bug code="UuF,UrF" />
+    <Bug code="UuF,UrF,RV" />
```
Comment (samgoldman) [inline exclusionBugs.xml:4]: Just curious, what's RV?

File: src/main/java/Main.java [hunk 2]
```diff
@@ -15,6 +21,18 @@
...
+            imageCache.put("3", new ImageView(new Image(ResourceLoader.thirdFloor.openStream())));
+            imageCache.put("2", new ImageView(new Image(ResourceLoader.secondFloor.openStream())));
+            imageCache.put("1", new ImageView(new Image(ResourceLoader.firstFloor.openStream())));
+            imageCache.put("L1", new ImageView(new Image(ResourceLoader.firstLowerFloor.openStream())));
+            imageCache.put("L2", new ImageView(new Image(ResourceLoader.secondLowerFloor.openStream())));
+            imageCache.put("G", new ImageView(new Image(ResourceLoader.groundFloor.openStream())));
+        } catch(IOException e) {
+            e.printStackTrace();
+        }
+        ApplicationState.getApplicationState().setImageCache(imageCache);
```
Comment (Nathanalphaman) [inline src/main/java/Main.java:35]: Are we sure we want to put image caching back in?

Comment (samgoldman) [reply src/main/java/Main.java:35]: He's doing it in the ApplicationState, which _should_ be better from a memory standpoint. I think the memory issues were related to the event bus issues that Maya found

Comment (Nathanalphaman) [reply src/main/java/Main.java:35]: Oh I didn't even notice that. I was talking to Maya and she said she will be ready to push her fix for that later tonight

**Review by samgoldman — APPROVED** [review]

**Pull request merged**